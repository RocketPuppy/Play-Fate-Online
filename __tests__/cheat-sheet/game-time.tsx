import * as React from "react";
import * as renderer from "react-test-renderer";
import GameTime from "../../src/cheat-sheet/game-time";

test("renders all the pieces", () => {
  const render = renderer.create(<GameTime />);

  expect(render.toJSON()).toMatchSnapshot();
});

