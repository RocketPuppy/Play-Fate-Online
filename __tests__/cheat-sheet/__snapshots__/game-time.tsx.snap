// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Game Time
  </h2>
  <dl>
    <dt>
      Exchange
    </dt>
    <dd>
      time for everyone to get a turn
    </dd>
    <dt>
      Scene
    </dt>
    <dd>
      time to resolve a situation
    </dd>
    <dt>
      Session
    </dt>
    <dd>
      a single sitting
    </dd>
    <dt>
      Scenario
    </dt>
    <dd>
      an episode
    </dd>
    <dt>
      Arc
    </dt>
    <dd>
      a season
    </dd>
    <dt>
      Campaign
    </dt>
    <dd>
      the entire game in a particular setting
    </dd>
  </dl>
</div>
`;
